web/gui/dashboard_info.js

```diff
@@ -570,6 +570,13 @@ netdataDashboard.menu = {
         icon: '<i class="fas fa-comments"></i>',
         info: 'Summary, namespaces and topics performance data for the <b><a href="http://pulsar.apache.org/">Apache Pulsar</a></b> pub-sub messaging system.'
     },
+
+    'alarms': {
+        title: 'Alarms',
+        icon: '<i class="fas fa-exclamation"></i>',
```

Comment:
Could be https://fontawesome.com/icons/bell?style=light or https://fontawesome.com/icons/bell-on?style=light

See https://fontawesome.com/icons?d=gallery&q=bell&m=pro

We have `bell` icon  for alarms in the top bar